tests/test_dgds_backend.py

```diff
@@ -13,33 +14,59 @@ def test_index(self):
         rv = self.client.get('/')
         self.assertIn('Welcome to DGDS', rv.data.decode())
 
-    def test_bad_datasetId(self):
-        response = self.client.get('/locations?locationCode=diva_id__270&datasetId=wrongcode')
-        result = json.loads(response.data.decode('utf-8'))
-        expected_output = {
-            'message': 'The provided datasetId does not exist'
-        }
-        self.assertEqual(expected_output, result)
+    @patch('dgds_backend.app.requests.get')
+    def test_get_wms_url(self, mock_get):
 
-    def test_get_datasets(self):
-        response = self.client.get('/datasets')
-        result = json.loads(response.data.decode('utf-8'))
+        mocked_fews_resp = '''{
+            "title": "Spatial Display",
+            "layers": [{
+                "name": "Wind NOAA GFS",
+                "title": "Wind NOAA GFS",
+                "groupName": "GLOSSIS",
+                "times": ["2019-08-01T10:00:00Z", "2019-08-01T13:00:00Z"]
+            }, {
+                "name": "Water Level",
+                "title": "Water Level",
+                "groupName": "D3D-FM gtsm",
+                "times": ["2019-08-01T12:00:00Z", "2019-08-01T13:00:00Z"]
+            }, {
+                "name": "Current 2DH",
+                "title": "Current 2DH",
+                "groupName": "D3D-FM gtsm",
+                "times": ["2019-08-01T12:00:00Z", "2019-08-01T13:00:00Z"]
+            }]
+        }'''
 
-        expected_output = {
-            "dataType": "timeseries",
-            "description": "To be filled by Daniel",
-            "id": "wl",
-            "name": "Waterlevel",
-            "timeSpan": "Live",
-            "units": "m",
-            "wmsUrl": "http://pl-tc012.xtr.deltares.nl:8080/FewsWebServices/wms?service=WMS&request=GetMap&version=1.3&layers=Water%20Level&styles=&format=image%2Fpng&transparent=true&crs=EPSG%3A3857&time=2019-04-24T10%3A00%3A00.000Z&uppercase=false&width=256&height=256&bbox={bbox-epsg-3857}"
-        }
-        self.assertIn(expected_output, result["Flooding"].get("datasets"))
-
-    def test_get_locations(self):
-        response = self.client.get('/locations?locationCode=diva_id__270&datasetId=wl')
-        result = json.loads(response.data.decode('utf-8'))
-        self.assertIn("geometry", result)
+        mock_get.return_value.status_code = 200
+        mock_get.return_value.text = mocked_fews_resp
+
+        id = "wd"
+        url_template = "http://pl-tc012.xtr.deltares.nl:8080/FewsWebServices/wms?service=WMS&request=GetMap&version=1.3&layers=Wind%20NOAA%20GFS&styles=&format=image%2Fpng&transparent=true&crs=EPSG%3A3857&time=##TIME##&uppercase=false&width=256&height=256&bbox={bbox-epsg-3857}"
```

Comment:
If this is only mocking, use the simplest url possible that's still testing something.